tests/pc/test_po_update.py

```diff
@@ -73,8 +75,8 @@ def test_po_update(duthost):
         time.sleep(30)
         int_facts = duthost.interface_facts()['ansible_facts']
         assert not int_facts['ansible_interface_facts'][portchannel]['link']
-        bgp_facts = duthost.bgp_facts()['ansible_facts']
-        assert bgp_facts['bgp_statistics']['ipv4_idle'] == 1
+        if not wait_until(120, 10, duthost.check_bgp_statistic, 'ipv4_idle', 1):
+            assert duthost.get_bgp_statistic('ipv4_idle') == 1
```

Comment:
Please use pytest_assert. assert sometimes triggers exception. (same for other asserts below)

Reply:
Done, thanks for the review.